packages/search/src/elasticsearch/dbhelper.test.ts

```diff
@@ -9,30 +9,53 @@
  * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
  * graphic logo are (registered/a) trademark(s) of Plan International.
  */
+// @ts-ignore
 import {
   indexComposition,
   updateComposition,
   searchForDuplicates,
   searchByCompositionId
 } from '@search/elasticsearch/dbhelper'
 import { mockCompositionBody } from '@search/test/utils'
-import { client } from '@search/elasticsearch/client'
 import { logger } from '@search/logger'
 import { IBirthCompositionBody } from '@search/elasticsearch/utils'
+import * as elasticsearch from '@elastic/elasticsearch'
+import { StartedElasticsearchContainer } from 'testcontainers'
+// @ts-ignore
```

Comment:
Can we somehow get rid of these ts-ignores? TS compiler doesn't seem to complain about these lines without the comments. You can use `yarn test:compilation` to check this.